style.css

```diff
@@ -1 +1,193 @@
 /*  Add your styling here */
+body {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  margin:0px;
+}
+
+.navbar {
+  display: none;
+}
+.nav {
+  display: none;
+}
+header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  background-color: #7b3f00;
+ 
+}
+.header {
+  width: 60%;
+}
+.sideimg {
+  width: 40%;
+}
+.headerimg {
+  width: 150px;
+  padding: 25px 0px 35px 25px;
+}
+.hamburger {
+  width: 30px;
+  position: absolute;
+  top: 30px;
+  right: 30px;
+}
+.sitemoto {
+  text-align: right;
+  padding: 100px 30px 0px 25px;
+  font-size: 1.2rem;
+  color: lightgoldenrodyellow;
+}
+img {
+  transition: 1sec;
+}
+img:hover {
+  transform: scale(0.97);
+}
+
+section {
+  padding: 10px;
+}
+.maintext1 {
+  font-size: 2rem;
+  font-weight: 100;
+  margin-bottom: 20px;
+}
+.maintext2 {
+  font-size: 1rem;
+  font-weight: 100;
+  text-align: justify;
+}
+.imgsection {
+  width:100%;
+  
+}
+
+.bottomsection {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.produce {
+  width: 70%;
+ 
+}
+
+footer {
+  background-color: #7b3f00;
+  color: lightgoldenrodyellow;
+  
+}
+a:link {
+  color: lightgoldenrodyellow;
+  text-decoration: none;
+  margin-bottom: 30px;
+}
+a:visited {
+  color: lightgoldenrodyellow;
+}
+li {
+  padding: 0px 20px 0px 20px;
+  list-style-type: none;
+  font-size: 1rem;
+}
```

Comment:
Ba carefuly here - this is styling all the `li`s on your page. You probably only want to target a specific list here - in this case, use a class.